Dockerfile

```diff
@@ -71,6 +71,8 @@ COPY --from=build /tmp/buildApp/dist .
 COPY ./config ./config
 COPY ./docker-entrypoint.sh .
 
+COPY ./resources ./resources
```

Comment:
you ignore the resources folder in .gitignore but copy it to the Dockerfile, what is the strcuture of the resources folder? what does it contain ?

Reply:
it was for development purposes, removed it.